.markdownlint-cli2.jsonc

```
@@ -194,6 +194,13 @@
           "searchPattern": ["/e-mail/ig"],
           "replace": ["email"],
           "skipCode": false
+        },
+        {
+          "name": "localhost-links",
+          "message": "Don't use localhost for links",
+          "searchPattern": "/\\]\\(https:\\/\\/localhost:\\d+\\//g",
```

```suggestion
          "searchPattern": "/\\]\\(https?:\\/\\/localhost:\\d+\\//g",
```

to catch http://localhost and https://localhost.